src/toolbox/GuidGenerator.ts

```diff
@@ -0,0 +1,27 @@
+//#region > Functions
+/**
+ * @function Generate random guid string.
+ * @returns {string} A guid string.
+ * @description This function generates a random guid string.
+ */
+function GuidGenerator() {
+  let d = new Date().getTime();
+
+  return "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx".replace(/[xy]/g, function(c) {
```

Comment:
Change this to a ES6 arrow function.
Ref: https://www.sitepoint.com/es6-arrow-functions-new-fat-concise-syntax-javascript/